src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -180,13 +180,15 @@ public String getImage() {
     private final String type;
     private final String dockerImage;
     private final List<Volume> volumes;
+    private final Boolean dockerPrivilegedMode;
 
     @DataBoundConstructor
-    public ContainerInfo(String type, String dockerImage, List<Volume> volumes)
+    public ContainerInfo(String type, String dockerImage, List<Volume> volumes, Boolean dockerPrivilegedMode)
       throws FormException {
```

Comment:
Usually IDE indent throw on new line left than internal code block to be more readable.